### PR TITLE
Update Setup/README.md with more accurate build instructions

### LINKED ISSUE #2259: The type or namespace name 'TeamFoundation' does not exist in the namespace 'Microsoft'
I'm trying to build gitextensions with Visual Studio Express 2013, but I get this error:

```
TfsHelper.cs(5,17): error CS0234: The type or namespace name 'TeamFoundation' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?) [C:\prog\gitextensions\Plugins\BuildServerIntegration\TfsInterop\TfsInterop.Vs2012.csproj]
```

I tried installing [Team Foundation Server 2013 Object Model Installer](http://visualstudiogallery.msdn.microsoft.com/3278bfa7-64a7-4a75-b0da-ec4ccb8d21b6) and [Team Explorer for Microsoft Visual Studio 2013](http://www.microsoft.com/en-us/download/details.aspx?id=40776), with no success.
What should I install?

### PR DESCRIPTION
**Pull request by RussKie**
This should resolve #2259.

Original work was done by @mindw in https://github.com/gitextensions/gitextensions/pull/2782

@KindDragon @jbialobr is there anything else worth adding?

### CODE REVIEW

File: Setup/README.md [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
+    git submodule init
+    git submodule sync
+    ```
+
+4. Run set_version_to.py (Python 2) to change current version. For example -
+    ```
+    set_version_to.py --version 2.50
+    ```
 
 2. Run set_version_to.py (Python 2) to change current version.
```
Comment (jbialobr) [inline Setup/README.md]: This is the same a 4.

File: Setup/README.md [hunk 2]
```diff
@@ -1,17 +1,32 @@
...
+
+3. Synchronize submodules:
+    ```
+    git submodule init
+    git submodule sync
+    ```
+
+4. Run set_version_to.py (Python 2) to change current version. For example -
+    ```
+    set_version_to.py --version 2.50
```
Comment (jbialobr) [inline Setup/README.md]: set_version_to.py -t 2.50 -v 2.50

File: Setup/README.md [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
 2. Run set_version_to.py (Python 2) to change current version.
 
-    set_version_to.py --version 2.48
+    set_version_to.py --version 2.50
 
 
-3. Run BuildInstallers.VS2012.cmd, or BuildInstallers.VS2013.cmd, or BuildInstallers.VS2015.cmd. This will generate two msi installers:
-    * GitExtensions248.msi - the standard install with only Git Extensions
-    * GitExtensions248Complete.msi - also includes Git for Windows and KDiff3
+3. Run BuildInstallers.VS2015.cmd or BuildInstallers.VS2017.cmd. This will generate two msi installers:
```
Comment (jbialobr) [inline Setup/README.md]: The numbering is inconsistent.

File: Setup/README.md (hunk 1 above)
Comment (RussKie) [inline Setup/README.md]: opps

**Comment by RussKie** [comment]
Fixed

**Pull request merged**